src/Modules/index.js

```diff
@@ -0,0 +1,4 @@
+export {default as pornhub} from './Pornhub';
+export {default as redtube} from './Redtube';
+export {default as sex} from './Sex';
+export {default as xvideos} from './Xvideos';
```

Comment:
Please add a newline at end and give spaces between brackets eg 
`export { ... } from ...`